metrics-clojure-ring/project.clj

```diff
@@ -1,8 +1,8 @@
-(defproject metrics-clojure-ring "2.10.0-SNAPSHOT"
+(defproject malesch/metrics-clojure-ring "2.9.1"
   :description "Various things gluing together metrics-clojure and ring."
   :url "https://github.com/sjl/metrics-clojure"
   :license {:name "MIT"}
-  :dependencies [[cheshire "5.7.0"]
-                 [metrics-clojure "2.10.0-SNAPSHOT"]]
+  :dependencies [[cheshire "5.7.1"]
```

Comment:
No worries. #123 is merged and I will try to produce a new release of `metrics-clojure` in the next couple of days.